### ISSUE TITLE
have you see the built in auto toollenght from 2.8 stglue ?

### ISSUE THREAD

**Issue by alkabal** [linuxcnc-probe-screen/probe-screen-ng#54]
https://github.com/seebaer1976/linuxcnc/blob/4ba291cf23b3a973feef15eb02148dcb5a6c9813/nc_files/remap_lib/python-stdglue/stdglue.py

**Comment by kiall** [comment]
Yes, it's based on the vers.by probe screen too - I don't think I've noticed any fixes in there. Is there something specific about it?

**Comment by alkabal** [comment]
no i have not reall read but it's a full python code replacing macro
